README.md

```diff
@@ -50,22 +50,62 @@ The following actions are supported:
 
 ### Virtual Machines / Servers
 
-* Create a new VM - `create_vm`
-* Reboot a VM - `reboot_vm`
-* Stop a VM - `stop_vm`
-* Start a VM - `start_vm`
-* Destroy a VM - `destroy_vm`
+* List available VMs - ``list_vms``
+* List available VM sizes - ``list_sizes``
+* List available VM images - ``list_images``
+* Create a new VM - ``create_vm``
+* Reboot a VM - ``reboot_vm``
+* Stop a VM - ``stop_vm``
+* Start a VM - ``start_vm``
+* Destroy a VM - ``destroy_vm``
+* Import public SSH key - ``import_public_ssh_key``
 
 ### Storage
 
-* Upload a file to a container - `upload_file`
+* Upload a file to a container - ``upload_file``
 * Enable CDN for a container and retrieve container CDN URL -
-  `enable_cdn_for_container`
-* Retrieve CDN URL of a CDN enabled container - `get_container_cdn_url`
+  ``enable_cdn_for_container``
+* Retrieve CDN URL of a CDN enabled container - ``get_container_cdn_url``
 * Retrieve CDN URL for an object which is stored in a CDN enabled container -
-  `get_object_cdn_url`
+  ``get_object_cdn_url``
 
 ### DNS
 
-* Create a new DNS record - `create_dns_record`
-* Delete an existing DNS record - `delete_dns_record`
+* Create a new DNS record - ``create_dns_record``
```

Comment:
Isn't the double backtick the syntax for RST and not Markdown?

Reply:
GitHub flavored markdown supports both notations - single and double backtick.